### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by Yaroslav-Stetsko — APPROVED** [review]
Looks good to me

File: templates/taxi/driver_confirm_delete.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car?</h1>
```
Comment (Nattalli) [inline templates/taxi/driver_confirm_delete.html:4]: Car?

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: templates/taxi/driver_confirm_delete.html (hunk 1 above)
Comment (HStakhiv) [reply templates/taxi/driver_confirm_delete.html:4]: Oh. I copied this file and forgot to change this :)

Comment (HStakhiv) [reply templates/taxi/driver_confirm_delete.html:4]: image

**Comment by HStakhiv** [comment]
Fix
image

**Comment by HStakhiv** [comment]
image

**Review by Nattalli — APPROVED** [review]
Okay, good job!

**Pull request closed without merging**